javascript/stricjs/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "@stricjs/router": "4.0.9"
+    "@stricjs/router": "4.0.10"
```

Comment:
I suggest using the last patch available

```suggestion
    "@stricjs/router": "~4.0.10"
```

Reply:
Thanks that looks good